### PR TITLE
Move authors out of list to Contributors at end of document

### PR DESCRIPTION
**Pull request by tfpauly**
Satisfies #184

### CODE REVIEW

File: draft-ietf-taps-arch.md [hunk 1]
```diff
@@ -431,3 +389,7 @@ This work has been supported by the UK Engineering and Physical Sciences
 Research Council under grant EP/R04144X/1.
 
 Thanks to Stuart Cheshire, Josh Graessley, David Schinazi, and Eric Kinnear for their implementation and design efforts, including Happy Eyeballs, that heavily influenced this work.
+
+# Contributors
+
+The authors of this document include Tommy Pauly, Brian Trammell, Anna Brunstrom, Godred Fairhurst, Colin Perkins, Philipp S. Tiesel, and Chris Wood.
```
Comment (britram) [inline draft-ietf-taps-arch.md]: I'd do this a bit differently (applies to all three docs)

- I'd place contributors above acknowledgments
- "include" sounds like there are other authors, which is not what we're saying.
- I haven't included editors in contributors' lists in the past "in addition to the editors, this document was written by:"
- I'd add back affiliations and (with author agreement, gdpr etc) email addresses
- and stylistically, I'd do this in list form.

Where specific sections have specific authors, this should also be noted (see 8095), but this is not the case here...

Comment (gorryfair) [reply draft-ietf-taps-arch.md]: Just to be sure ...

Are you suggesting keeping the current list of authors in the boilerplate; diving text between contributors and acknowledgment sections. - That I would agree with.

If you were to be suggesting pruning the authors list - I would not want that for the document s were I am currenty listed as an author: specifically I *would* wish to be in the RFC-Ed last call for these documents.

Comment (mwelzl) [reply draft-ietf-taps-arch.md]: See my answer to this in #184: I agree with Gorry here. Specifically, as long as we're not told that we MUST do such a thing, I think we should try to keep the author list as-is.

**Comment by britram** [comment]
+1 @mwelzl I would also be okay with not doing this, and fighting the RPC about it. 

(One of the reasons the limit is being advertised more forcefully is that there have been instances where authorship doesn't necessarily reflect work done, in some cases because people are financially rewarded for getting their name on an RFC or even an Internet-Draft, and you have people who really have no clue what is in the document on the critical path in auth48. This is not the case here.)

(Doubleparenthetical: someone should have a talk with the RSE about how to handle actual large collaborations such as this one where not everyone needs to be on the RPC last call, but where people actually should show up in the metadata that gets pulled into various academically-important indices.)

**Comment by mwelzl** [comment]
I hope we all agree that this can be closed; it's probably better addressed with an email to the chairs.

**Comment by tfpauly** [comment]
Sure, I don't particularly care—I don't see why it's a huge problem, considering that the QUIC documents, etc, are doing this style, but if you guys want to fight the man, that's fine too =)

**Pull request closed without merging**